src/client/main.py

```diff
@@ -5,6 +5,22 @@
 
 
 def send_data(data):
```

Comment:
Looks great! As we talked, we should switch to using a config file for the server details.

Reply:
thank you, and yes thats a good idea